flatliners/baseflatliner.py

```diff
@@ -41,6 +42,12 @@ def cluster_id(self, x):
     def cluster_version(self, x):
         return self.metric_label(x, 'version')
 
-
-
-
+    @staticmethod
+    def create_cache_dict(maxsize=500, ttl=600):
+        # Here max size is determined by the number of clusters whose
```

Comment:
that's not true. this just creates a cached dict. depends on the caller what he wants to store in this dict

Reply:
`max size is determined` *Default value was determined*

Reply:
I mean this method doesnt know anything about the number of clusters... so dont put a reference in here.